packages/bruno-js/src/bruno-request.js

```diff
@@ -56,11 +56,31 @@ class BrunoRequest {
     this.req.headers[name] = value;
   }
 
-  getBody() {
+  /**
+   * Get the body of the request
+   * 
+   * @param {*} options 
+   * @param {boolean} options.raw - If true, return the raw body without parsing it
+   * @returns 
+   */
+  getBody(options = {}) {
```

Comment:
What about `req.body` that is set inside the constructor? I think this should also be JSON parsed like req.getBody

Reply:
💯 good catch (this could have resulted in breakages for folks who were accessing `req.body` in scripts)

Have updated the code with a comment

```javascript
constructor(req) {
    // existing code

    /**
     * We automatically parse the JSON body if the content type is JSON
     * This is to make it easier for the user to access the body directly
     * 
     * It must be noted that the request data is always a string and is what gets sent over the network
     * If the user wants to access the raw data, they can use getBody({raw: true}) method 
     */
    const isJson = hasJSONContentType(this.req.headers);
    if (isJson) {
      this.body = this.__safeParseJSON(req.data);
    }
  }
```